src/main/resources/liquibase/workbench_changelog/12_5_0.xml

```diff
@@ -0,0 +1,713 @@
+			PRIMARY KEY (role_id, permission_id),
+			INDEX fk_role_permission_2_idx (permission_id ASC),
+			CONSTRAINT fk_role_permission_1
+			FOREIGN KEY (role_id)
+			REFERENCES role (id)
+			ON DELETE NO ACTION
+			ON UPDATE NO ACTION,
+			CONSTRAINT fk_role_permission_2
+			FOREIGN KEY (permission_id)
+			REFERENCES permission (permission_id)
+			ON DELETE NO ACTION
+			ON UPDATE NO ACTION);
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-18">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="role_type_permission"/>
+			</not>
+		</preConditions>
+		<comment>
+			Create table role_type_permission
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+			CREATE TABLE role_type_permission (
+			role_type_id INT(11) NOT NULL,
+			permission_id INT(11) NOT NULL,
+			selectable TINYINT(1) NULL,
+			PRIMARY KEY (role_type_id, permission_id),
+			INDEX fk_role_type_permission_2_idx (permission_id ASC),
+			CONSTRAINT fk_role_type_permission_1
+			FOREIGN KEY (role_type_id)
+			REFERENCES role_type (role_type_id)
+			ON DELETE NO ACTION
+			ON UPDATE NO ACTION,
+			CONSTRAINT fk_role_type_permission_2
+			FOREIGN KEY (permission_id)
+			REFERENCES permission (permission_id)
+			ON DELETE NO ACTION
+			ON UPDATE NO ACTION);
+		</sql>
+	</changeSet>
+
+	<changeSet author="vanina" id="v12.5.0-19">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="0">
+				select count(1)
+				from permission
+			</sqlCheck>
+		</preConditions>
+		<comment>
+			INSERT INTO table permission
+		</comment>
+		<sql dbms="mysql" splitStatements="true">
+        	 INSERT INTO permission (permission_id, name, description, parent_id, workbench_sidebar_category_link_id) values (1, 'ADMIN','Full',null,null);
+			 INSERT INTO permission (permission_id, name, description, parent_id, workbench_sidebar_category_link_id) values (2, 'ADMINISTRATION','Administration',1,null);
+			 INSERT INTO permission (permission_id, name, description, parent_id, workbench_sidebar_category_link_id) values (3, 'CROP_MANAGEMENT','Crop Management',1,null);
+			 INSERT INTO permission (permission_id, name, description, parent_id, workbench_sidebar_category_link_id) values (4, 'BREEDING_ACTIVITIES','Breeding Activities',1,null);
+			 INSERT INTO permission (permission_id, name, description, parent_id, workbench_sidebar_category_link_id) values (5, 'INFORMATION_MANAGEMENT','Information Management',1,null);
+			 INSERT INTO permission (permission_id, name, description, parent_id, workbench_sidebar_category_link_id) values (6, 'STATISTICAL_ANALYSIS','Statistical Analysis',1,null);
+			 INSERT INTO permission (permission_id, name, description, parent_id, workbench_sidebar_category_link_id) values (7, 'SITE_ADMIN','Site Admin',2,null);
+			 INSERT INTO permission (permission_id, name, description, parent_id, workbench_sidebar_category_link_id)
+			 select 8, 'MANAGE_ONTOLOGIES','Manage Ontologies',3,
+				 IFNULL( ( select  sidebar_category_link_id
```

Comment:
I don't see the value of enclosing the subqueries for getting **sidebar_category_link_id** per too in `IFNULL `clause when we choose to insert null anyway. I think the `IFNULL` is unnecessary.